static/sass/styles.scss

```diff
@@ -1323,3 +1323,16 @@ hr.p-separator.is-shallow {
 .u-aspect-ratio--3-2 {
   aspect-ratio: 3/2;
 }
+
+// XXX Pete
+// Custom styles for paper background
+.is-paper {
+  background: #f3f3f3;
```

Comment:
Is this significantly different from `p-strip--light`?

Reply:
As I understand it, this is to be applied to the whole page without the extra styling that comes with a p-strip. @lyubomir-popov will be able to explain in more detail